explore_utils.py

```diff
@@ -1,5 +1,22 @@
 import numpy as np
 
+def revenue_per_trafficsource(dataset):
+    """Find the average transavtion revenue by traffic Source
+
+    args :
+         dataset (Dataset): the google analytics Dataset
+
+    returns:
+         Dataframe of average transaction revenue by traffic Source
+    """
+    train_df = dataset.train
+    train_df['revenue'] = train_df['totals.transactionRevenue'].astype(float)
```

Comment:
This will add a new column to train_df that is reflected for everyone in `explore.py`. Please refer to the comments I posted in PR #49 for instructions on how to test and fix this.